app/js/ContactManager.js

```diff
@@ -67,7 +67,7 @@ module.exports = ContactManager = {
     )
   }
 }
-;['touchContact', 'getContacts'].map(method =>
+;['touchContact', 'getContacts'].map((method) =>
```

Comment:
Oh good, I always disliked prettier's previous default of removing the parens here for single argument arrow functions 👍